packages/web/package.json

```diff
@@ -13,9 +13,9 @@
     "lint:fix": "prettier --write \"**/*\" && next lint --fix",
     "pre-commit": "lint-staged",
     "generate": "yarn generate:chain-info && yarn generate:wallet-assets && yarn generate:wallet-registry",
-    "generate:chain-info": "ts-node --esm config/generate-chain-infos/index.ts",
-    "generate:wallet-assets": "ts-node --esm config/generate-wallet-assets/index.ts",
-    "generate:wallet-registry": "ts-node --esm config/generate-wallet-registry.ts"
+    "generate:chain-info": "dotenv -e .env -e .env.local -- ts-node --esm config/generate-chain-infos/index.ts",
```

Comment:
Are we sure `dotenv` doesn't pull in all env files by default, without having to specify them?

There could be many varieties:
```
# local env files
.env.local
.env.development.local
.env.test.local
.env.production.local
```